requestor-tutorials/vm-runtime/computation-payload-manifest.md

```diff
@@ -0,0 +1,278 @@
+---
+description: Computation Payload Manifest description, its schema, and configuration guide
+---
+
+# Computation Payload Manifest
+
+_Computation Payload Manifest_ allows [Requestor](../../introduction/requestor.md) to define application package _Payload_ and allows to set constraints on _Computations_ performed on [Provider](../../introduction/provider.md) node.
+
+Request with a manifest can be [configured in yapapi](#configuration). 
+
+Provider node operator controls what computation can be performed by:
+
+  - [Importing certificates](#3-importing-application-authors-certificates) used to sign App authors' certificates into Provider's [keystore](../../provider-tutorials/provider-cli.md#keystore) (which allows to verify _manifest_ signature)
+
+  - Adding domain patterns to Provider's [domain whitelist](../../provider-tutorials/provider-cli.md#domain-whitelist) (which makes [signature](#2-manifest-signature) optional _manifests_)
+
+## Configuration
+
+Manifest can be configured as a [yapapi.payload.vm.manifest](https://yapapi.readthedocs.io/en/latest/api.html#module-yapapi.payload.manifest) function parameter together with optional manifest signature and [App author's certificate](#certificates).
+
+Check [external API request example](../service-development/service-example-6-external-api-request.md) for details.
+
+## Manifest schema
+
+_Computation Payload Manifest_ must follow a specific [JSON Schema](computation-payload-manifest.schema.json) ([Documentation](computation-payload-manifest.schema.md))
+
+### Schema verification
+
+Manifest can be verified using `jsonschema` library:
+
+```sh
+wget https://handbook.golem.network/requestor-tutorials/vm-runtime/computation-payload-manifest.schema.json
+pip install jsonschema
+jsonschema --instance manifest.json computation-payload-manifest.schema.json
+```
+
+### Payload object
+
+_Computation Payload Manifest_ **must** contain at least one _Payload_ object. 
+
+_Payload_ definition allows to define [GVMI images](convert-a-docker-image-into-a-golem-image.md) used by Application and supported architecture.
+
+Simple _Computation Payload Manifest_ with _Payload_ definition:
+
+```json
+{
+  "version": "0.1.0",
+  "createdAt": "2022-09-15T00:00:00.000000Z",
+  "expiresAt": "2100-01-01T00:00:00.000000Z",
+  "payload": [
+    {
+      "platform": {
+        "arch": "x86_64",
+        "os": "linux"
+      },
+      "urls": [
+        "http://girepo.dev.golem.network:8000/docker-golem-hello-world-latest-779758b432.gvmi"
+      ],
+      "hash": "sha3:d646d7b93083d817846c2ae5c62c72ca0507782385a2e29291a3d376"
+    }
+  ]
+}
+```
+
+### Computation Manifest object
```

Comment:
This partially duplicates content of autogenerated from code comments `computation-payload-manifest.schema.md`. I am not sure if we want to have this duplication.